README.md

```diff
@@ -24,236 +25,21 @@ We do not collect any personal information. If you wish to disable telemetry
-- Specify the account uploading and instantiating the contract with `--suri`, which in this example is the default
-  development account of `//Alice`.
-  For other accounts, the actual secret key must be provided e.g. an 0x prefixed 64 bit hex string, or the seed phrase.
-
-> :warning: **Use only for development**: Use a safer method of signing here before using this feature with production
-> projects. We will be looking to provide alternative solutions in the future!
-
-- You also can specify the url of your node with `--url ws://your-endpoint`, by default it is
-  using `ws://localhost:9944`.
-- To perform a dry-run via RPC to estimate the gas usage without submitting a transaction use the `--dry-run` flag.
-
-For more information about the options,
-check [cargo-contract documentation](https://github.com/paritytech/cargo-contract/blob/master/crates/extrinsics/README.md#instantiate)
-
-Interacting with the Smart Contract:
-
-1. Read-only Operations: For operations that only require reading from the blockchain state. This approach does not
-   require to submit an extrinsic.
-   Example using the get() message:
-
-```sh
-pop call contract -p ./my_contract --contract $INSTANTIATED_CONTRACT_ADDRESS --message get --suri //Alice
-```
-
-2. State-modifying Operations: For operations that change a storage value, thus altering the blockchain state. Include
-   the `x / --execute`  flag to submit an extrinsic on-chain.
-
-Example executing the `flip()` message:
-
-```sh
-pop call contract -p ./my_contract --contract $INSTANTIATED_CONTRACT_ADDRESS --message flip --suri //Alice -x
-```
-
-## E2E testing
-
-For end-to-end testing you will need to have a Substrate node with `pallet contracts`.
-You do not need to run it in the background since the node is started for each test independently.
-To install the latest version:
-```
-cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
-```
-
-Run e2e testing on the Smart Contract:
-
-```sh
-# Run e2e tests for an existing smart contract
- pop test contract  -p ./my_contract --features e2e-tests
-```
-
-If you want to run a different node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable using the  `--node` flag:
-```sh
-# Run e2e tests for an existing smart contract
- pop test contract  -p ./my_contract --features e2e-tests --node YOUR_CONTRACTS_NODE_PATH
-```
-
-### Pallets
-
-To create a new Pallet, simply run `pop new pallet`. You will have a new pallet ready for hacking.
-To customize the new Pallet you can follow these options:
-
-```sh
-# create a pallet with name `pallet-awesome` in the current working directory
-pop new pallet pallet-awesome
-# or with options
-pop new pallet pallet-awesome --authors Me --description "This pallet oozes awesomeness" --path my_app/pallets
-```
+Pop CLI guides for **appchains**:
+- https://learn.onpop.io/v/appchains
 
 ## Building Pop CLI locally
```

Comment:
Optional: 
I saw `cargo-contracts` has a section `Installation using Docker Image` (https://github.com/use-ink/cargo-contract?tab=readme-ov-file#installation-using-docker-image). We can add it too to provide more options to devs

Reply:
Where can I find the steps to install Pop CLI using a Docker image?

Reply:
Let's add this in a separate PR.